evennia/objects/objects.py

```diff
@@ -886,7 +894,7 @@ def logerr(string="", err=None):
             # check if source location lets us go
             try:
                 if source_location and not source_location.at_pre_object_leave(
-                    self, destination, **kwargs
+                        self, destination, **kwargs
```

Comment:
Why 8 character indent here, four should be enough... no big deal though; we'll run `black` on it ... but it's a bit weird since it creates so big diffs.